packages/app-config-writer/src/smartlinks-config/generateSmartLinks.ts

```diff
@@ -0,0 +1,27 @@
+import { create as createStorage } from 'mem-fs';
+import { create } from 'mem-fs-editor';
+import type { Editor } from 'mem-fs-editor';
+import type { ToolsLogger } from '@sap-ux/logger';
+import type { TargetConfig } from '../types';
+import { writeSmartLinksConfig } from './utils';
+
+/**
+ * @description Add smartlinks configuration to a UI5 application.
```

Comment:
Just noticed the `@description` tag you used. We never use it and put the main description as plain text at the beginning. Could you please to the same for your jsdoc comments.

Reply:
Removed `description` tag - initially added because of local eslint issues

Reply:
Hmm - I am running into linting issues also on the pipeline. Seems like [`jsdoc/match-description`](https://github.com/SAP/open-ux-tools/blob/main/.eslintrc#L126) throws errors (except in tests).
Should that be changed?

Reply:
Why would this happen just in your change and nowhere else?  Please check the linting issues closely, maybe you are not matching the description regex correctly.

Reply:
You are right, closing dot missing. Working on commit